ow_plexil/src/plexil-adapter/OwInterface.h

```diff
@@ -176,9 +176,17 @@ class OwInterface : public PlexilInterface
 
   FaultMap64 m_systemErrors =
   {
+    {"NONE", std::make_pair(0,false)},
```

Comment:
As is, "NONE" is reported as a fault when the node starts.  Initializing to true solves this.  But I'm not sure that NONE is a good entry to have in this map because of its inverse semantics from the other entries.  Is it properly managed?  Does it serve any purpose?

Reply:
Good catch, this was an erroneous addition when I was updating names. I've removed this entry.

I also replaced the literals with the appropriate constants from the message definition.